sdks/python/apache_beam/runners/portability/portable_runner_test.py

```diff
@@ -235,6 +234,20 @@ def create_options(self):
     return options
 
 
+# TODO(BEAM-7248): Delete this test after PortableRunner supports
+# beam:runner:executable_stage:v1.
+@unittest.skip("BEAM-7248")
```

Comment:
are you intending to have this test skipped? I think it should not be skipped, right?

Reply:
Yes, I misplaced the `unittest.skip`. This test should run and the test _with_ fusion should be skipped.